CHANGELOG.md

```diff
@@ -102,6 +102,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1424](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1424))
 - Remove db.name attribute from Redis instrumentation
   ([#1427](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1427))
+- `opentelemetry-instrumentation-system-metrics` Fix initialization of the instrumentation class when configuration is provided
```

Comment:
This should be in ## Unreleased >> ## Fixed section

Reply:
Good point! Updated.